curriculum/challenges/english/blocks/workshop-bookstore-page/69134b6cb43aa5f254950a10.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 69134b6cb43aa5f254950a10
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+
+# --description--
+
+Start your bookstore page by creating the HTML boilerplate.
+
+Add the `<!DOCTYPE html>` declaration and the opening `<html>`, `<head>`, and `<title>` tags.
+
+Set the page title to `XYZ Bookstore Page`.
+
+# --hints--
+
+Your code should include a `<!DOCTYPE html>` declaration.
+
+```js
+assert.match(code, /<!DOCTYPE html>/i);
+```
+
+Your code should include `<html>` and <head> tags.
+
+```js
+assert.match(code,/<html/i);
+assert.match(code,/<head/i);
+```
+
+Your <title> element should be XYZ Bookstore Page.
+
+```js
+assert.match(code,/<title>XYZ Bookstore Page<\/title>/);
+```
+
+# --seed--
+## --seed-contents--
+
+```html
+--fcc-editable-region--
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>XYZ Bookstore Page</title>
+</head>
+--fcc-editable-region--
```

Comment:
the seed content is what the step start with, it should not start with the things you ask to include